torch/optim/optimizer.py

```diff
@@ -393,6 +394,27 @@ def pack_group(group):
             'param_groups': param_groups,
         }
 
+    @staticmethod
+    def _process_value_according_to_param_policy(param: Tensor, value: Tensor, paramID: int = None,
```

Comment:
tiny nit: might be more pythonic to name as `param_id` rather than `paramID` and similarly for other variables ([see here](https://peps.python.org/pep-0008/#function-and-variable-names))